### ISSUE TITLE
Panic

### ISSUE THREAD

**Issue by gregorst3** [kulupu/kulupu#11]
```
 Hash: given=a317706fffc970902b71c5df6ef46171bd16c6e9880788426870a0c5897078bb, expected=7fabbf5438e5a7af4c42ddddc2386e09655854725c781aed1de61bb8c04b6830 `

 ====================

Version: 0.2.0-0f0c1a3-x86_64-linux-gnu

stack backtrace:
   0: substrate_panic_handler::set::{{closure}}
   1: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:477
   2: std::panicking::begin_panic
   3: frame_executive::Executive<System,Block,Context,UnsignedValidator,AllModules>::execute_block
   4: kulupu_runtime::api::dispatch
   5: std::panicking::try::do_call
   6: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:80
   7: substrate_executor::native_executor::safe_call
   8: std::thread::local::LocalKey<T>::with
   9: <kulupu::service::Executor as substrate_executor::native_executor::NativeExecutionDispatch>::dispatch
  10: std::thread::local::LocalKey<T>::with
  11: substrate_state_machine::StateMachine<B,H,N,T,Exec>::execute_aux
  12: substrate_state_machine::StateMachine<B,H,N,T,Exec>::execute_call_with_native_else_wasm_strategy
  13: substrate_state_machine::StateMachine<B,H,N,T,Exec>::execute_using_consensus_failure_handler
  14: <substrate_client::call_executor::LocalCallExecutor<B,E> as substrate_client_api::call_executor::CallExecutor<Block,substrate_primitives::hasher::blake2::Blake2Hasher>>::call_at_state
  15: substrate_client::client::Client<B,E,Block,RA>::apply_block
  16: <&substrate_client::client::Client<B,E,Block,RA> as substrate_consensus_common::block_import::BlockImport<Block>>::import_block
  17: <alloc::sync::Arc<T> as substrate_consensus_common::block_import::BlockImport<B>>::import_block
  18: substrate_consensus_common::import_queue::import_single_block
  19: <futures_util::future::poll_fn::PollFn<F> as core::future::future::Future>::poll
  20: futures_util::future::future::chain::Chain<Fut1,Fut2,Data>::poll
  21: <futures_util::future::poll_fn::PollFn<F> as core::future::future::Future>::poll
  22: futures_executor::thread_pool::PoolState::work
  23: std::sys_common::backtrace::__rust_begin_short_backtrace
  24: std::panicking::try::do_call
  25: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:80
  26: core::ops::function::FnOnce::call_once{{vtable.shim}}
  27: <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once
             at /rustc/4560ea788cb760f0a34127156c78e2552949f734/src/liballoc/boxed.rs:922
  28: <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once
             at /rustc/4560ea788cb760f0a34127156c78e2552949f734/src/liballoc/boxed.rs:922
      std::sys_common::thread::start_thread
             at src/libstd/sys_common/thread.rs:13
      std::sys::unix::thread::Thread::new::thread_start
             at src/libstd/sys/unix/thread.rs:79
  29: start_thread
  30: clone


Thread 'import-queue-worker-0' panicked at 'Storage root must match that calculated.', vendor/substrate/frame/executive/src/lib.rs:297`

This is a bug. Please report it at:

        https://github.com/kulupu/kulupu/issues
```

**Comment by gregorst3** [comment]
Tried to purge-chain , upgrade substrate but same situation
Kulupu 0.2.1

**Comment by gregorst3** [comment]
Fixed in 0.2.2
